src/identity/models/image.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+module.exports = (sequelize, DataTypes) => {
```

Comment:
Shouldn't we just export an arrow function here, that way we can just import it and export it as a module here

https://github.com/charge-tech/paycertify/blob/dev/services/svc-identity-server/src/infra/database/models/person.js